lighthouse-treemap/app/src/main.js

```diff
@@ -641,10 +666,41 @@ function injectOptions(options) {
   document.head.append(scriptEl);
 }
 
+/**
+ * @param {import('../../../lighthouse-core/lib/i18n/locales').LhlMessages} localeMessages
+ */
+function getStrings(localeMessages) {
+  const strings = /** @type {Record<string, string>} */ ({});
+  for (const icuMessageId of Object.keys(localeMessages)) {
+    const [filename, varName] = icuMessageId.split(' | ');
+    if (!filename.endsWith('util.js')) throw new Error(`Unexpected message: ${icuMessageId}`);
+
+    const key = /** @type {keyof LH.I18NRendererStrings} */ (varName);
+    strings[key] = localeMessages[icuMessageId].message;
+  }
+
+  return strings;
+}
+
 /**
  * @param {LH.Treemap.Options} options
  */
 function init(options) {
+  const i18n = new I18n(options.lhr.configSettings.locale, {
+    // Set missing renderer strings to default (english) values.
+    ...TreemapUtil.UIStrings,
+    ...getStrings(locales[options.lhr.configSettings.locale]),
```

Comment:
does this handle all that fallback business? I can't remember off the top of my head if the `configSettings` locale is normalized to always match our defined locale (e.g. for user-provided `de-CH` which matches our `de` key, will this work or miss?)

EDIT: seems like we should be fine https://github.com/GoogleChrome/lighthouse/blob/f06807bb7fb61a94b75b0b836075351bf8985f33/lighthouse-core/config/config-helpers.js#L319